src/content/1.1/code/ocaml/snippet02.ml

```diff
@@ -1,6 +1 @@
-module type Polymorphic_Function_G = sig
```

Comment:
It was definitely confusing for me when I started going through the ocaml book (as an ocaml beginner when I saw these modules). :+1: